ActindoWorkFlow/UserInterface/02a_Workflows.md

```diff
@@ -0,0 +1,313 @@
+
+    - ![Unpublish](/Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
+  Click this entry to unpublish the view. This menu entry is only displayed when a view is selected and published.
+
+    - ![Save](/Assets/Icons/Save.png "[Save]") save  
+  Click this entry to save the current view settings in the selected view. This menu entry is only displayed when a view is selected.
+
+      > [Info] When the settings of a view have been changed, an asterisk is displayed behind the view name. The asterisk is hidden as soon as the changes have been saved.
+
+    - ![Delete](/Assets/Icons/Trash01.png "[Delete]") delete  
+  Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed when a view is selected.
+
+
+
+- ![Search](/Assets/Icons/Search.png "[Search]") (Search)  
+    Click this button to display the search bar and search for a workflow.
+
+    > [Info] The search function can be used to search for a workflow name and key.
+
+- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+    Click this button to update the list of workflow versions.
+
+- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)  
+    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The x indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](/Assets/Icons/Filter.png "[Filter]") Filter (x)  
+    Click this button to display the filter bar and customize the active filters. The x indicates the number of filters that are currently active.
+
+
+The list displays all workflow versions of the selected workflow. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Version*  
+  Version number of the workflow.
+
+- *Published*  
+  - ![Check](/Assets/Icons/Check.png "[Check]") (Checkmark): The workflow version is published.
+  - ![Cross](/Assets/Icons/Cross02.png "[Cross]") (Cross): The workflow version is unpublished.
+
+- *ID*  
+  Workflow ID.
+
+
+- [x]  
+Select the checkbox to display the editing toolbar.
+
+- [PUBLISH]/[UNPUBLISH]  
+Click this button to publish or unpublish a workflow version. For detailed information, see [Publish a workflow](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#publish-a-workflow) and [Unpublish a workflow](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#unpublish-a-workflow).
+
+- [DELETE]  
+Click this button to delete the selected workflow version.  
+
+  > [Info] To prevent any problems, it is strongly recommended not to delete a workflow. For detailed information, see [Delete a workflow](ActindoWorkFlow/Operation/01_ManageWorkflows.md#delete-a-workflow).
+
+- [VIEW]  
+Click this button to display the selected workflow version.
+
+[comment]: <> (Oli: Button DELETE ist i.d.R. ein "Trash" Symbol. Button VIEW funktioniert nicht)
```

Comment:
verschieben in Zeile 169/170-> zwischen Filter button und "The list displays all workflow versions of the selected workflow..."